paymentsheet/src/main/java/com/stripe/android/paymentelement/EmbeddedPaymentElement.kt

```diff
@@ -77,6 +77,17 @@ class EmbeddedPaymentElement @Inject internal constructor(
         return configurationCoordinator.configure(intentConfiguration, configuration)
     }
 
+    /**
+     * A composable function that displays a vertical list of wallet payment methods that can be used for express
+     * checkout.
+     */
+    @WalletsButtonPreview
```

Comment:
These names don't align. WalletsButton vs WalletButtons.

I actually think `WalletButtons` make more sense. Can we change the annotation?